src/test/codegen/consts.rs

```diff
@@ -10,11 +10,11 @@
 // CHECK: @STATIC = {{.*}}, align 4
 
 // This checks the constants from inline_enum_const
-// CHECK: @{{[0-9]+}} = {{.*}}, align 2
+// CHECK: @alloc5 = {{.*}} zeroinitializer, align 2
```

Comment:
Just realized the `zeroinitializer` shouldn't be here either.

Reply:
https://github.com/rust-lang/rust/pull/69155/commits/4b2ee4594bb4bc9e1c6be1af8b8284063e62f8b9